dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceApplication.java

```diff
@@ -190,6 +191,10 @@ public class DockstoreWebserviceApplication extends Application<DockstoreWebserv
     public static final String GA4GH_API_PATH_V2_BETA = "/api/ga4gh/v2";
     public static final String GA4GH_API_PATH_V2_FINAL = "/ga4gh/trs/v2";
     public static final String GA4GH_API_PATH_V1 = "/api/ga4gh/v1";
+    private static final List<String> CORS_ENDPOINTS = Arrays.asList(
+            GA4GH_API_PATH_V2_BETA + "/*",
```

Comment:
How do the "extended" api's factor into this? 
https://dockstore.org/api/static/swagger-ui/index.html#/extendedGA4GH

Reply:
As far as I know we don't have an active use case for serving these to another web app, but this will allow it. I think that we want to support people using the extended APIs via the browser.

Reply:
Yeah, to keep things simple, they can all be protected. It would be neat to allow one or two through (the organisation ones?) but I'd be fine with just treating them all the same as proprietary